tailwind.config.js

```diff
@@ -13,14 +13,20 @@ module.exports = {
             ],
             serif: ['Noto Serif TC', '-apple-system', 'serif'],
         },
+        backgroundSize: {
+            auto: 'auto',
+            cover: 'cover',
+            contain: 'contain',
+            bgsize: '7%,5%,7%,7%,7%,7%,7%,5%,7%,7%,7%,7%',
```

Comment:
`bgsize` is a vague term. Everything under `backgroundSize` is considered a bgsize...
Can you elaborate more on this? Name things in a more self-explanatory manner.

Reply:
It make sense. thanks.